app/assets/stylesheets/narrow-table-display.scss

```diff
@@ -1,33 +1,68 @@
-th.table-combine_right_if_narrow .table-hidden_right_title {
+th.table-combine_right_if_narrow .table-hidden_right_title,
+th.table-combine_right_if_narrow_2 .table-hidden_right_title {
```

Comment:
What is the significance of the `2`? That's a magic number! What is it about these `td` elements that means they need to be classified differently? Could an additional flag be used to identify the elements that need the special styling. For example, say the special condition is they are made of chocolate. `th` elements that need to be combined if narrow could be classified as:

- **Made of chocolate** `table-combine_right_if_narrow made_of_chocolate`
- **Made of something else** `table-combine_right_if_narrow`

With this the `table-combine_right_if_narrow` would still apply to the chocolate elements, and styling applied just to them could be applied to `.made_of_chocolate` or even `.table-combine_right_if_narrow.made_of_chocolate`

Reply:
I agree that the `2` might not be the best label.  Perhaps something else.  

The issue here is that pretty much the entire styling has to be separate because it all has to be in their respective @media queries.  So there is very little that can be combined - even though the styles are pretty much identical.  

So having `table-combine_right_if_narrow made_of_chocolate` will mean that the `table-combine_right_if_narrow` class virtually redundant.  (The style at line 3 is the only one that won't be reliant on `made_of_chocolate`.)

I am thinking a class `table-combine_right_if_narrow_at_555px` might be a good way of doing it, but that means changing the class name every time the width changes - as indeed it did as part of this change.  

Unless SASS has a clever way of dealing with identical styles coming into play at different widths, then I don't think these can be improved (minus, perhaps, my choice of class name).

Reply:
I have changed the CSS class names:

- Removing the `2` class
- Adding new class `narrow_555` for the 555px class
- Adding new class `narrow_470` for the 470px class

Reply:
Those class names are very specific but I can't think of better alternatives (`narrow` and `slightly_narrower` :smile: ). So lets go for `narrow_555` and `narrow_470`